src/test/resources/posters/tests/unit/LoginTest.json

```diff
@@ -2,14 +2,14 @@
 	{
 		"firstName": "John",
 		"lastName": "Doe",
-		"email": "john@varmail.net",
+		"email": "john@doe.com",
 		"password": "topsecret",
 		"testId": "happy path"
 	},
 	{
 		"firstName": "John",
 		"lastName": "Doe",
-		"email": "john@varmail.net",
+		"email": "john@doe.com",
```

Comment:
didn't we agree on varmail accounts?

Reply:
The database makes this account by default, as a login-able account. I'll try and see if I can make it so that the db for posters creates a varmail acc instead of a john@doe.com.

Reply:
Our login now is at admin@example.com.